Maple2.Server.World/Migrations/20230911002445_UgcMarket.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Maple2.Server.World.Migrations
+{
+    /// <inheritdoc />
+    public partial class UgcMarket : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropIndex(
+                name: "IX_ugcresource_OwnerId",
+                table: "ugcresource");
+
+            migrationBuilder.AddColumn<byte>(
+                name: "Type",
+                table: "ugcresource",
+                type: "tinyint unsigned",
+                nullable: false,
+                defaultValue: (byte)0);
+
+            migrationBuilder.AddColumn<int>(
+                name: "TabId",
+                table: "ugc-market-item",
+                type: "int",
+                nullable: false,
+                defaultValue: 0);
+
+            migrationBuilder.AddColumn<bool>(
+                name: "Giftable",
+                table: "premium-market-item",
+                type: "tinyint(1)",
+                nullable: false,
+                defaultValue: false);
+
+            migrationBuilder.AddColumn<string>(
```

Comment:
missing `defaultValue`, this is needed for all `AddColumn` to avoid breaking old rows.